CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ endif()
 
 # User options.
 option(OPENMP "use OpenMP threading" ON)
+option(LINK_W3EMC "Link w3emc library" ON)
```

Comment:
Not sure the purpose of this new option. Shouldn't this build in the same manner on all platforms? Have the other platforms been tested with this change?

Reply:
This change is required for building with spack-stack-1.8.0. In spack-stack-1.8.0, the skgb function has been added to the g2 library and will throw an error for multiple definitions (it is also in w3emc). Once gfs_utils gets updated to spack-stack-1.8.0 everywhere, the option can be removed.

Reply:
Yes please

Reply:
@WalterKolczynski-NOAA Added to #88 and updated the issue ticket.